examples/icd_code_prediction/icd_coding.py

```diff
@@ -0,0 +1,107 @@
+import sys
+import yaml
+from termcolor import colored
+
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.readers import PlainTextReader
+from forte.pipeline import Pipeline
+from forte.processors.writers import PackIdJsonPackWriter
+
+from ft.onto.base_ontology import (
+    Token,
+)
+from fortex.spacy import SpacyProcessor
+
+# import MedicalArticle to get article data from data pack
```

Comment:
These import comments seem superfluous, since the name itself implies what's being imported.